doc/classes/EditorSettings.xml

```diff
@@ -117,20 +144,55 @@
 				Returns the value of the setting specified by [param name]. This is equivalent to using [method Object.get] on the EditorSettings instance.
 			</description>
 		</method>
+		<method name="get_shortcut" qualifiers="const">
+			<return type="Shortcut" />
+			<param index="0" name="path" type="String" />
+			<description>
+				Returns the shortcut specified by [param path]. If no shortcut with the provided path is found in the shortcut list, tries to find a built-in action. If found, adds it to the list and returns it, otherwise returns [code]null[/code].
```

Comment:
```suggestion
				Returns the shortcut specified by [param path]. Tries to find a built-in action if no shortcut with the provided path is found in the shortcut list. If found, adds it to the list and returns it, otherwise returns [code]null[/code].
```